### PR TITLE
fix issue with pdf upload in cloud

### PR DESCRIPTION
**Pull request by dusvyat**
## Description

fix issue with pdf upload in cloud

loom video: https://www.loom.com/share/f1174519df794f9b8312a8cd5f5a81e9

## Type of change

(Please delete options that are not relevant)

- [x] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] 📄 This change requires a documentation update

## Verification Process

To ensure the changes are working as expected:

 - [x]   Test Location: dev.mindsdb.com
 - [ ]   Verification Steps: tested pdf upload in UI

## Additional Media:

- [x] I have attached a brief loom video or screenshots showcasing the new functionality or change.

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have appropriately commented on my code, especially in complex areas.
- [ ] Necessary documentation updates are either made or tracked in issues.
- [x] Relevant unit and integration tests are updated or added.

### CODE REVIEW

File: requirements/requirements.txt [hunk 1]
```diff
@@ -31,6 +31,7 @@ pytz
 botocore
 boto3
 python-dateutil
+pymupdf
```
Comment (dusvyat) [inline requirements/requirements.txt:34]: this is already included in `mindsdb_gateway` config.json fyi

**Comment by dusvyat** [comment]
@hamishfagg any way to get around this one, failing on requirement check as `pymupdf` is imported via an alias `fit`z. Therefore, when I add` pymupdf`  to main requirements, the check fails as it doesn’t find it being imported in the code base as alias is used.

**Review by paxcema — APPROVED** [review]
Looks good

**Comment by hamishfagg** [comment]
For future reference you can map between a package name and import name here: https://github.com/mindsdb/mindsdb/blob/aa3078b94010283d4eeb4f0f12b10c8ed7a508bf/tests/scripts/check_requirements.py#L62

There was already a mapping for fritz somehow, so just needed to move pymupdf to the file handler requirements file

**Pull request merged**